WebUI/templates/Replicate.html

```diff
@@ -29,6 +45,9 @@
 
       <th>Movement</th>
       <th>Running Time</th>
+
+      {% if localURL == '/worthToNotice' %}<th>Action</th>{% endif %}
+
```

Comment:
When using page templates you shouldn't be checking what the local URL is. This indicates that something in the code needs to be refactored - most likely the "Long Running Replicates" page should have it's own HTML page. Remember, while code reuse is good, it shouldn't interfere with the clarity of the code.